### ISSUE TITLE
NVDA/JAWS is not announcing shortcut key for Results and message tab: A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_ScreenReader

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17876]
## Actual:
NVDA/JAWS is not announcing shortcut key for Results and Message tab.

## Expected:
NVDA/JAWS should announce shortcut key for Results and Message tab.

## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG1.3.1;#A11yMAS;#Win32;#InfoAndRelationships;#ScreenReader;#JAWS;#NVDA;#FTP;#STP;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code: 1.86.2
Extension Version: v1.22.1
ScreenReader: JAWS(2023.2212.23), NVDA(2023.1.3)

## Repro Steps:
1. Turn on NVDA/JAWS
2. Open Visual studio code
3. Tab till extension button and search for "mssql".
4. Tab till install and move to the SQL Server button.
5. Tab till Add Connection Button and insert the valid credentials.
6. Tab till the database and hit enter.
7. Tab till the System database and hit enter.
8. Tab till the Tables and hit enter.
9. Tab till any table under System tables and hit shift + F10
10. Tab till Select Top 1000 and hit enter
11. Tab till Execute Query and hit enter
12. Tab till "Results" dropdown and verify whether screen reader is announcing shortcut key or not

## User Impact:
Screen reader user will not be able to get the shortcut key if screen reader does not announce it.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![Presentation3](https://github.com/microsoft/vscode-mssql/assets/100269580/d0e4582f-108e-40f0-9016-b23d9bf1063f)
![Presentation4](https://github.com/microsoft/vscode-mssql/assets/100269580/2592d8f9-81cf-40c0-8ee2-e738bb50ea73)

**Comment by laurennat** [comment]
fixed in this PR: https://github.com/microsoft/vscode-mssql/pull/17935

**Comment by aniketr99** [comment]
Regressed the bug in the latest environment.
Issue is fixed. So, closing the bug. Attachment is added below for the reference.
NVDA fixed:
![Screenshot (547)](https://github.com/user-attachments/assets/8bbb0819-47d5-4cd0-afc6-12e758218eef)

JAWS Fixed:
![Screenshot (549)](https://github.com/user-attachments/assets/33886516-95f7-44e1-b1f6-774e6a513b10)
![Screenshot (548)](https://github.com/user-attachments/assets/4ac189e3-ca9a-40a1-accb-5a9108da2aae)
